index.html

```diff
@@ -13,39 +13,254 @@
+                    fill="#5E6C84" />
+                </svg></a>
+              </li>
+              <div class="language language-body">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse 1.png" alt="">
+                </div>
+                <span>JavaScript</span>
+              </div>
+              <div class="language language-body">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse_2.png" alt="">
+                </div>
+                <span>HTML</span>
+              </div>
+              <div class="language language-body">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse_3.png" alt="">
+                </div>
+                <span>CSS</span>
+              </div>
+              <li class="language-item framework-skills">
+                <h1>Frameworks</h1>
+                <a href="#"><svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
+                    fill="#505F79" />
+                </svg></a>
+              </li>
+              <li class="language-item framework-skills">
+                <h1>Skills</h1>
+                <a href=""><i class="fa-solid fa-angle-right"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+  </section>
+  <section class="contact-me">
+      <div class="contact-header">
+        <h2 class="contact-title">Contact Me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+          needs
+          coding. I’d love to help with it</p>
+      </div>
+      <form class="contact-form" action="https://formspree.io/f/xqkjglvp" method="post">
+        <div class="input-field">
+          <input type="text" class="input" placeholder="Something..." required maxlength="30">
+        </div>
+        <div class="input-field">
+          <input type="email" class="input" placeholder="someone@example.com" required>
+        </div>
+        <div class="input-field">
+          <textarea class="textarea" placeholder="Write something here..." maxlength="500" required></textarea>
+        </div>
+        <div class="input-field">
+          <input type="submit" class="btn" value="Get In Touch">
+        </div>
+        <div class="bottom-rectangle"></div>
+      </form>

```

Comment:
- [ ] The data is not submitted appropriately, kindly work on it so that the data will drop in your mail box.
![submit](https://user-images.githubusercontent.com/86200356/199422401-5ba0c5b9-5e7c-4b0a-8a5a-67dfbaee3d09.png)
- [ ] Kindly wrap the form in a footer tag because it is actually in the footer(lowest part of the page).